### PR TITLE
feat: freeze first column - iOS

### PR DESCRIPTION
**Pull request by vcellu**
This is for [MD-3366](https://jira.qlikdev.com/browse/MD-3366)
Note: you will get a warning when reszing the first column on frozen one.   The issue is that the size of the frame is updated prior to the getting the size of the cell.  And things go out of synch.  But this is a benign warning.

### CODE REVIEW

File: ios/DataCollectionView.swift [hunk 1]
```diff
@@ -135,31 +186,34 @@ class DataCollectionView: UIView, UICollectionViewDataSource, UICollectionViewDe
...
+  //MARK: collectionview
   func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
     // swiftlint:disable:next force_cast
     let cell = collectionView.dequeueReusableCell(withReuseIdentifier: reuseIdentifier, for: indexPath) as! DataCellView
     cell.isDataView = self.isDataView
     cell.backgroundColor = isDataView ? indexPath.row % 2 == 0 ? .white : UIColor(red: 0.97, green: 0.97, blue: 0.97, alpha: 1.0) : .white
     cell.cellColor = cellColor
     cell.numberOfLines = cellStyle.rowHeight ?? 1
-    if let data = dataRows, let columnWidths = columnWidths {
+    if let data = dataRows, let columnWidths = columnWidths, let dataColumns = dataColumns {
```
Comment (gruskal) [inline ios/DataCollectionView.swift:197]: `let columnWidths = columnWidths, let dataColumns = dataColumns` ?

File: ios/MasterColumnCollectionView.swift [hunk 2]
```diff
@@ -8,35 +8,30 @@
...
+       withRows rows: [DataRow],
        andColumns cols: [DataColumn],
        theme: TableTheme,
        selectionsEngine: SelectionsEngine,
        cellStyle: CellContentStyle,
        columnWidths: ColumnWidths) {
-    super.init(frame: frame, withRows: rows, andColumns: cols, theme: theme, selectionsEngine: selectionsEngine, cellStyle: cellStyle, columnWidths: columnWidths)
-    fitToFrame()
+    super.init(frame: frame,
+               withRows: rows,
```
Comment (gruskal) [inline ios/MasterColumnCollectionView.swift:19]: Spacing seems off here. We should setup a linter before we push the alpha branch to master

**Review by gruskal — APPROVED** [review]

File: ios/DataCollectionView.swift (hunk 1 above)
Comment (vcellu) [reply ios/DataCollectionView.swift:197]: yes, common practice in Swift.

File: ios/MasterColumnCollectionView.swift (hunk 2 above)
Comment (vcellu) [reply ios/MasterColumnCollectionView.swift:19]: we have a lintel... I'll run it

**Pull request merged**